components/footer/footer.sass

```diff
@@ -0,0 +1,38 @@
+.footer 
+  background: $light-neutral-color
+  height: 15vh
+  width: 100%
+  display: flex
+  align-items: center
+  justify-content: space-evenly
+
+.footer__logo
+  height: 100%
+
+.footer__links
+  display: flex
+  flex-direction: column
+  align-items: center
+  justify-content: center
+  height: 100%
+  width: 33%
+
+.footer__link
+  color: $dark-neutral-color
+  transition: color .3s
+
+.footer__link:hover
+  color: $footer-text-hover-color
+
+.footer__item
+  color: $dark-neutral-color
+  margin: 0.5rem
+  font-weight: 300
+
+.footer__info
+  display: flex
+  flex-direction: column
+  align-items: center
+  justify-content: center
+  height: 100%
+  width: 33%
```

Comment:
How's this handle on very small screens? Should 33% be a max-width with a wider base width?
Just a question, if it handles good on small screens leave as is :)

Reply:
Actually, just realized they're completely redundant with the new changes, thanks for pointing out